docker/edge/http/edge_stop.sh

```diff
@@ -0,0 +1,2 @@
+docker stop edgebroker && docker rm $_
```

Comment:
mention in the start of the script what should be input prameter and what this script is designed for.

Reply:
Comment Incorporated